sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusMessageBatch.cs

```diff
@@ -91,7 +96,9 @@ public bool TryAddMessage(ServiceBusMessage message)
             lock (_syncGuard)
             {
                 AssertNotLocked();
-                return InnerBatch.TryAddMessage(message);
+
```

Comment:
Regarding that discussion, should we add the documentation here to detail the behavior or did you have another place in mind?